inference/README.md

```diff
@@ -4,26 +4,25 @@ The triton backend for shark.
 
 # Build
 
-Install dSHARK
+Install SHARK
```

Comment:
I made these edits this morning. Just make sure you are on top of main

Reply:
It isn't reporting any merge conflicts though? is it fine to just merge it?